iac/tf-multienv-cicd-anthos-autopilot/variables.tf

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project_id" {
+    type = string
+    description = "Project ID where the resources will be deployed"
```

Comment:
We could do this, but I'm more leaning towards not doing this because:
- this is not a multi-cloud repo, which means there is little value in adding this prefix
- CFTK terraform modules always use `project_id`, terraform modules use `project` as the key for this value e.g.

```
module "enabled_google_apis" {
  ...
  project_id                  = var.project_id
  ...
}
resource "google_artifact_registry_repository" "container_registry" {
  ...
  project       = var.project_id
  ...
}

```

If you still prefer `google_cloud_project_id`, I'm happy to change it.